docker/Makefile.template

```diff
@@ -14,7 +14,6 @@ build: Dockerfile
 push: build
 	# Requires ~/.dockercfg
 	docker push ${name}:${tag}
-	docker push ${name}:latest
```

Comment:
It's down to your preference - I'm really just copying the Toil appliance build methodology.